adapter/internal/oasparser/envoyconf/listener.go

```diff
@@ -246,6 +247,53 @@ func CreateVirtualHosts(vhostToRouteArrayMap map[string][]*routev3.Route) []*rou
 			Domains: []string{vhost, fmt.Sprint(vhost, ":*")},
 			Routes:  routes,
 		}
+		conf, _ := config.ReadConfigs()
+		if conf.Envoy.RateLimit.Enabled {
+			virtualHost.RateLimits = []*routev3.RateLimit{
```

Comment:
This is not getting applied for /health, /ready right? as we are not setting dynamic metadata?

Reply:
Reverted from vhost level to route level as envoy does not throttle in this setup.